packages/backend/prisma/migrations/20220905163559_add_order_table/migration.sql

```diff
@@ -0,0 +1,21 @@
+-- CreateEnum
+CREATE TYPE "OrderStatus" AS ENUM ('CREATED', 'PAID', 'DELIVERED', 'CANCELLED');
+
+-- CreateTable
+CREATE TABLE "Order" (
+    "id" UUID NOT NULL,
+    "productId" UUID NOT NULL,
+    "buyerId" UUID NOT NULL,
+    "cost" MONEY NOT NULL,
```

Comment:
What currency will be stored? USD or UAH

Reply:
@voonya, Will use hryvna for now